README.md

```diff
@@ -20,19 +20,20 @@ Prerequisites
 -------------
 
 
-### Linux
+### Linux and Mac OS 
 
 For Ginkgo core library:
 
-*   _cmake 3.1+_
+*   _cmake 3.8+_
 *   C++11 compliant compiler, one of:
-    *   _gcc 5.4.0+_
-    *   _clang 3.3+_ (__TODO__: verify, works with 5.0)
+    *   _gcc 5.5.0+_
+    *   _clang 3.9+_
+    *   _Apple LLVM 8.0+_ (__TODO__: verify)
 
 The Ginkgo CUDA module has the following __additional__ requirements:
 
 *   _cmake 3.10+_
```

Comment:
Not sure why we write 3.10+ here when we only require 3.8 in the CMakeFile.

Reply:
I was finally able to test it with 3.8 and it does work properly. I will update that.

Reply:
Thanks!